src/data/init/init-common-anims.ts

```diff
@@ -2,15 +2,14 @@ import { LegacyAnimConfig } from "#animations/anim-config";
 import { commonAnims } from "#animations/common-anims";
 import { globalScene } from "#app/global-scene";
 import { CommonAnim } from "#enums/common-anim";
-import { getTSEnumKeys, getTSEnumValues } from "#utils/common-utils";
 
 export function initCommonAnims(): Promise<void> {
   return new Promise((resolve) => {
-    const commonAnimNames = getTSEnumKeys(CommonAnim);
-    const commonAnimIds = getTSEnumValues(CommonAnim);
+    const commonAnimNames = Object.keys(CommonAnim);
+    const commonAnimIds = Object.values(CommonAnim);
     const commonAnimFetches: Promise<Map<CommonAnim, LegacyAnimConfig>>[] = [];
     for (let ca = 0; ca < commonAnimIds.length; ca++) {
-      const commonAnimId = commonAnimIds[ca];
+      const commonAnimId: CommonAnim = commonAnimIds[ca];
```

Comment:
_See previous comment:_
> You can avoid the const commonAnimId: CommonAnim = ... type definition by providing a type argument here

```suggestion
      const commonAnimId = commonAnimIds[ca];
```